site/content/docs/5.2/getting-started/parcel.md

```diff
@@ -1,101 +1,152 @@
+   ```
+
+3. **Install Bootstrap.** Now we can install Bootstrap. We'll also install Popper since our dropdowns, popovers, and tooltips depend on it for their positioning. If you don't plan on using those components, you can omit Popper here.
+
+   ```sh
+   npm i --save bootstrap @popperjs/core
+   ```
+
+Now that we have all the necessary dependencies installed and setup, we can get to work creating the project files and importing Bootstrap.
+
+## Project structure
+
+We've already created the `my-project` folder and initialized npm. Now we'll also create our `src` folder, stylesheet, and JavaScript file to round out the project structure. Run the following from `my-project`, or manually create the folder and file structure shown below.
+
+```sh
+mkdir {src,src/js,src/scss}
+touch src/index.html src/js/main.js src/scss/styles.scss
+```
+
+When you're done, your complete project should look like this:
 
 ```text
-project-name/
-├── build/
-├── node_modules/
-│   ├── @popperjs/
-|   |   └── core/
-│   └── bootstrap/
-├── scss/
-│   └── custom.scss
+my-project/
 ├── src/
-│   ├── index.html
-│   └── index.js
+│   ├── js/
+│   │   └── main.js
+│   ├── scss/
+│   │   └── styles.scss
+│   └── index.html
+├── package-lock.json
 └── package.json
 ```
 
-## Importing JavaScript
+At this point, everything is in the right place, but Parcel needs an HTML page and npm script to start our server.
 
-Import [Bootstrap's JavaScript]({{< docsref "/getting-started/javascript" >}}) in your app's entry point (usually `src/index.js`). You can import all our plugins in one file or separately if you require only a subset of them.
+## Configure Parcel
 
-<!-- eslint-skip -->
-```js
-// Import all plugins
-import * as bootstrap from 'bootstrap';
+With dependencies installed and our project folder ready for us to start coding, we can now configure Parcel and run our project locally. Parcel itself requires no configuration file by design, but we do need an npm script and an HTML file to start our server.
 
-// Or import only needed plugins
-import { Tooltip as Tooltip, Toast as Toast, Popover as Popover } from 'bootstrap';
+1. **Fill in the `src/index.html` file.** Parcel needs a page to render, so we use our `index.html` page to setup some basic HTML, including our CSS and JavaScript files.
 
-// Or import just one
-import Alert as Alert from '../node_modules/bootstrap/js/dist/alert';
-```
+   ```html
+   <!doctype html>
+   <html lang="en">
+     <head>
+       <meta charset="utf-8">
+       <meta name="viewport" content="width=device-width, initial-scale=1">
+       <title>Bootstrap w/ Parcel</title>
+       <link rel="stylesheet" href="scss/styles.scss">
+       <script type="module" src="js/main.js"></script>
+     </head>
+     <body>
+       <div class="container py-4 px-3 mx-auto">
+         <h1>Hello, Bootstrap and Parcel!</h1>
+         <button class="btn btn-primary">Primary button</button>
+       </div>
+     </body>
+   </html>
+   ```
```

Comment:
"We're including a little bit of Bootstrap styling here with the `div class="container"` and `<button>` so that we see when Bootstrap's CSS is loaded by Parcel." could be included as well after this HTML code as in Webpack guide.